editor/code_editor.cpp

```diff
@@ -1414,8 +1414,6 @@ void CodeTextEditor::set_edit_state(const Variant &p_state) {
 			text_editor->set_line_as_bookmark(bookmarks[i], true);
 		}
 	}
-
-	text_editor->grab_focus();
```

Comment:
This will need testing with folded lines and function list, as in `_members_overview_selected` we reset the scroll. Don't know if this will break the focus.